<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>767</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>2</horstretch>
    <verstretch>2</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>548</width>
    <height>767</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>zamnapp</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>531</width>
      <height>144</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>15</number>
     </property>
     <property name="topMargin">
      <number>15</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>15</number>
     </property>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="url_input">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>22669</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="placeholderText">
        <string>please enter url</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="search_button">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="from_spotify">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="specialValueText">
        <string/>
       </property>
       <property name="prefix">
        <string>Start Position </string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1999</number>
       </property>
       <property name="singleStep">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="output_area">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>541</width>
      <height>421</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>50</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::VLine</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>537</width>
       <height>417</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="check_all">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>Check All</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="download_frame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QRadioButton" name="res360_downlaod">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>360p</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="download_button">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>57</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>91</width>
             <height>0</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>255</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>255</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Light">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>170</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>170</red>
                 <green>255</green>
                 <blue>127</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>170</red>
                 <green>255</green>
                 <blue>127</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>255</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>255</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Light">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>170</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>170</red>
                 <green>255</green>
                 <blue>127</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>170</red>
                 <green>255</green>
                 <blue>127</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>120</red>
                 <green>120</green>
                 <blue>120</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>255</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Light">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>120</red>
                 <green>120</green>
                 <blue>120</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>170</red>
                 <green>255</green>
                 <blue>127</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>170</red>
                 <green>255</green>
                 <blue>127</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Download</string>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="shortcut">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="audio_download">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>160kpbs</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="res720_download">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>720p</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QProgressBar" name="download_progress">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QComboBox" name="which_download">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string>YouTube Track</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string comment="0">YouTube Track</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="1">YouTube Playlist</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="2">YouTube search</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="3">Spotify Track</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="4">Spotify Playlist</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="loading_gif">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>which_download</tabstop>
  <tabstop>url_input</tabstop>
  <tabstop>search_button</tabstop>
  <tabstop>output_area</tabstop>
  <tabstop>check_all</tabstop>
  <tabstop>res360_downlaod</tabstop>
  <tabstop>res720_download</tabstop>
  <tabstop>audio_download</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
